code/modules/clothing/suits/armor.dm

```diff
@@ -537,6 +537,28 @@
 	stiffness = LIGHT_STIFFNESS
 	style = STYLE_NONE
 
+/obj/item/clothing/suit/storage/vest/technomancer_old
+	name = "reinforced Technomancer armor"
+	desc = "Technomancer League's ballistic armor, less protective against industrial hazards but better in a fight"
+	icon_state = "armor_engineering_old"
+	item_state = "armor_engineering_old"
+	blood_overlay_type = "armor"
+	armor = list(
+		melee = 8,
+		bullet = 12,
+		energy = 7,
+		bomb = 40,
+		bio = 0,
+		rad = 10
```

Comment:
These stats are better in almost every way than the insulated suit.

Reply:
It has the stats of the reinforced helmet

Reply:
Then it needs some nerfs

Reply:
what should the stats of the reinforced techno gear be?

Reply:
> These stats are better in almost every way than the insulated suit.

it doesn't have bio or rad protection and less bomb. and isn't shockproof

Reply:
Rad protection lower than 100 matters very little (you will get toxins no matter 1 or 10 rad), I don't see existing armor having BIO armor and siemens_coefficient on armor matters very little (they wear insulated anyway)
Do you really think people will take more "useful" armor for their job instead of what will actually protect them in a firefight?

Reply:
What would you suggest I do to the stats then? Buff insulated or nerf reinforced?

Reply:
Nerf reinforced. You could give it more slowdown or something like worse stats. Right now it's strictly better

Reply:
it has been nerfed, asked people for stats suggestion on discord and this was the consensus